tcp_tests/managers/common_services_manager.py

```diff
@@ -12,18 +12,22 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-class CommonServicesManager(object):
+from tcp_tests.managers.execute_commands import ExecuteCommandsMixin
+
+
+class CommonServicesManager(ExecuteCommandsMixin):
     """docstring for CommonServicesManager"""
 
-    __config = None
-    __underlay = None
+    _config = None
```

Comment:
Please keep these variables private. If you need to use them in ExecuteCommandsMixin , please pass 'underlay' and 'config' objects to ExecuteCommandsMixin.__init__ and store them as private for this class